xbmc/addons/kodi-addon-dev-kit/include/kodi/kodi_adsp_types.h

```diff
@@ -465,6 +457,35 @@ extern "C" {
   /*!
    * @brief Structure to transfer the methods from kodi_audiodsp_dll.h to KODI
    */
+
+  typedef struct AddonProps_AudioDSP
+  {
+    const char* strUserPath;                    /*!< @brief path to the user profile */
+    const char* strAddonPath;                   /*!< @brief path to this add-on */
+  } AddonProps_AudioDSP;
+
+  typedef AddonProps_AudioDSP AE_DSP_PROPERTIES;
+
+  typedef struct AddonToKodiFuncTable_AudioDSP
+  {
+    KODI_HANDLE kodiInstance;
+
+    void (*AddMenuHook)(void* kodiInstance, AE_DSP_MENUHOOK *hook);
+    void (*RemoveMenuHook)(void* kodiInstance, AE_DSP_MENUHOOK *hook);
+    void (*RegisterMode)(void* kodiInstance, AE_DSP_MODES::AE_DSP_MODE *mode);
+    void (*UnregisterMode)(void* kodiInstance, AE_DSP_MODES::AE_DSP_MODE *mode);
+
+    ADSPHANDLE (*SoundPlay_GetHandle)(void* kodiInstance, const char *filename);
+    void (*SoundPlay_ReleaseHandle)(void* kodiInstance, ADSPHANDLE handle);
+    void (*SoundPlay_Play)(void* kodiInstance, ADSPHANDLE handle);
+    void (*SoundPlay_Stop)(void* kodiInstance, ADSPHANDLE handle);
+    bool (*SoundPlay_IsPlaying)(void* kodiInstance, ADSPHANDLE handle);
+    void (*SoundPlay_SetChannel)(void* kodiInstance, ADSPHANDLE handle, AE_DSP_CHANNEL channel);
+    AE_DSP_CHANNEL (*SoundPlay_GetChannel)(void* kodiInstance, ADSPHANDLE handle);
+    void (*SoundPlay_SetVolume)(void* kodiInstance, ADSPHANDLE handle, float volume);
+    float (*SoundPlay_GetVolume)(void* kodiInstance, ADSPHANDLE handle);
```

Comment:
Just as info this Sound Play become removed in furture and separated to independent header to have on all addons (a bit reworked).